securesystemslib/gpg/common.py

```diff
@@ -714,9 +712,18 @@ def parse_signature_packet(data, supported_signature_types=None,
   # conflict resolution scheme that makes more sense.
   # (see RFC4880 5.2.4.1.)
   # Below we only consider the last and favor hashed over unhashed subpackets
-  # TODO: Should we warn if a we use an unhashed subpacket?
-  for subpacket_type, subpacket_data in \
-      unhashed_subpacket_info + hashed_subpacket_info:
+  for idx, subpacket_tuple in \
+      enumerate(unhashed_subpacket_info + hashed_subpacket_info):
+
+    is_hashed = (idx >= len(unhashed_subpacket_info))
```

Comment:
As an outsider, it's a little unclear to me how `is_hashed` is verified. I think a comment is warranted on why the index is being compared to the length.

Reply:
Let me know if 9906acd works for you.